src/scripts/inc/common.sh

```diff
@@ -45,14 +45,15 @@ function init_systemd {
 function create_user {
 
     echo -n "Creating a user to run AlexaPi under ... "
-    UID_TEST=`id -u ${RUN_USER} >/dev/null 2>&1`
+    UID_TEST=$(id -u ${RUN_USER} >/dev/null 2>&1)
     UID_TEST="$?"
 
     if [ ${UID_TEST} -eq 0 ]; then
```

Comment:
```
if id -u ${RUN_USER} >/dev/null 2>&1; then
...
```
This shall work and does not create subshell and don't use special variables like `$?`